### PR TITLE
[KEYCLOAK-7681] Review pax-web OSGi dependencies

### PR DESCRIPTION
**Pull request by grgrzybek**

### CODE REVIEW

File: distribution/adapters/osgi/features/src/main/resources/features.xml [hunk 1]
```diff
@@ -79,8 +79,9 @@
     <!-- Keycloak adapter for PaxWeb undertow provider -->
     <feature name="keycloak-pax-http-undertow" version="${project.version}">
         <details>Keycloak Pax-Web adapter for Undertow</details>
-        <feature>keycloak-osgi-adapter</feature>
         <feature>keycloak-adapter-core</feature>
+        <feature version="[7.1,9)">pax-web-core</feature>
```
Comment (hmlnarik) [inline distribution/adapters/osgi/features/src/main/resources/features.xml]: lines 71 and 96 should be fixed in the same manner

Comment (grgrzybek) [reply distribution/adapters/osgi/features/src/main/resources/features.xml]: you mean these commented ones?

Comment (hmlnarik) [reply distribution/adapters/osgi/features/src/main/resources/features.xml]: yes, they're going to be open in community soon

**Comment by grgrzybek** [comment]
#5305 is fine. closing this one

**Pull request closed without merging**